docs/guides/slurm-hpc-ux.mdx

```diff
@@ -0,0 +1,165 @@
+
+HPC applications use the Slurm QPU resources assigned to the Slurm job.
+
+Environment variables provide more details for use by the application, e.g. `SLURM_JOB_QPU_RESOURCES` listing the quantum resource names (comma separated if there are several provided).
+These variables will be used by QRMI. (See the README files in the various QRMI directories ([ibm](https://github.com/qiskit-community/qrmi/blob/main/examples/qiskit_primitives/ibm/README.md), [pasqal](https://github.com/qiskit-community/qrmi/blob/main/examples/qiskit_primitives/pasqal/README.md)) for more details.)
+
+```python
+from qiskit import QuantumCircuit
+# using an IBM QRMI flavor:
+from qrmi.primitives import QRMIService
+from qrmi.primitives.ibm import SamplerV2, get_target
+
+# define circuit
+
+circuit = QuantumCircuit(2)
+circuit.h(0)
+circuit.cx(0, 1)
+circuit.measure_all()
+
+# instantiate QRMI service and get quantum resource (we'll take the first one should there be serveral of them)
+# inject credentials needed for accessing the service at this point
+load_dotenv()
+service = QRMIService()
+
+resources = service.resources()
+qrmi = resources[0]
+
+# Generate transpiler target from backend configuration & properties and transpile
+target = get_target(qrmi)
+pm = generate_preset_pass_manager(
+    optimization_level=1,
+    target=target,
+)
+
+isa_circuit = pm.run(circuit)
+
+# run the circuit
+options = {}
+sampler = SamplerV2(qrmi, options=options)
+
+job = sampler.run([(isa_circuit, isa_observable, param_values)])
+print(f">>> Job ID: {job.job_id()}")
+
+result = job.result()
+print(f">>> {result}")
+```
+
+See the [examples directory](https://github.com/qiskit-community/qrmi/tree/main/examples/qiskit_primitives/) for example files.
+
+### Backend specifics
+#### IBM Direct Access API
+##### Administrator scope
+Configuration of Direct Access API backends (HPC admin scope) includes endpoints and credentials to the Direct Access endpoint, authentication services as well as the S3 endpoint.
+Specifically, this includes:
+
+* IBM Cloud API key for creating bearer tokens
+* endpoint of Direct Access API
+* S3 bucket and access details
+
+Access credentials should not be visible to HPC users or other non-privileged users on the system.
+Therefore, sensitive data can be put in separate files which can be access protected accordingly.
+
+Note that Slurm has got full access to the backend.
+This has several implications:
+
+* the Slurm plugin is responsible for multi-tenancy (ensuring that users don't see results of other users' jobs)
+* vetting of users (who is allowed to access the QPU) and ensuring according access is up to the HPC cluster side
+* the capacity and priority of the QPU usage is solely managed through Slurm; there is not other scheduling of users involved outside of Slurm
+
+##### User scope
+Execution lanes are not exposed to the HPC administrator or user directly.
+Instead, mid term, there can be two different modes that HPC users can specify:
```

Comment:
```suggestion
Instead, mid-term, there can be two different modes that HPC users can specify:
```
I'm not sure what mid-term refers to here (and once previously in this file) - does it mean "currently, but will probably change, because it's under development"?